dove/src/stdout/print.rs

```diff
@@ -0,0 +1,64 @@
+use crate::stdout::STDOUT_STREAM;
+
+/// Prints output.
+#[macro_export]
+macro_rules! stdout {
+        ($($args:tt)*) => {
+            $crate::stdout::print::print(&format!($($args)*));
+        };
+    }
+
+/// Prints output with a newline.
+#[macro_export]
+macro_rules! stdoutln {
+        () => ($crate::stdout::print::print("\n"));
+        ($e:expr) => ($crate::stdout::print::println($e));
+        ($($args:expr),*) => (
+            $crate::stdout::print::println(&format!($($args),*));
+        )
+    }
+
+/// Print output.
+pub fn print(text: &str) {
+    STDOUT_STREAM
+        .get()
+        .unwrap()
```

Comment:
I don't belive we should leave unwraps in production code. Please use `expect` with some message, like `expect("Stdout stream is not initialized")`. This allows to quickly react on user's issues without asking for backtrace.